packages/react/test/specs/components/Popup/Popup-test.tsx

```diff
@@ -266,4 +263,44 @@ describe('Popup', () => {
       expect(contentElement.classList.contains(Popup.Content.className)).toEqual(true)
     })
   })
+
+  describe('keyboard event propagation', () => {
+    const expectPopupToStopPropagation = (
+      behavior: Accessibility,
+      shouldStopPropagation: boolean,
+    ) => {
+      const popup = mountWithProvider(
+        <Popup
+          trigger={<span id={triggerId}> text to trigger popup </span>}
+          content={<span id={contentId} />}
+          accessibility={behavior}
+        />,
+      )
+
+      // check popup open on key press
+      const popupTriggerElement = popup.find(`#${triggerId}`)
+      popupTriggerElement.simulate('keydown', { keyCode: keyboardKey.Enter })
+
+      expect(getPopupContent(popup).exists()).toBe(true)
```

Comment:
Not sure that we need this assertion there.

Reply:
you're right, will remove